CHANGELOG.md

```diff
@@ -26,9 +26,11 @@
     - theme
   - Removed isolate name from Dart context. It's now reported via the threads interface. It can be enabled via `options.attachThreads`
 - Use `sentryClientName` instead of `sdk.identifier` ([#1135](https://github.com/getsentry/sentry-dart/pull/1135))
+- Refactor `BindingUtils` to `BindingWrapper` to enable the use of custom bindings ([#1184](https://github.com/getsentry/sentry-dart/pull/1184))
```

Comment:
If this is a refactor, we don't need an entry here, it's more about user-facing changes.

Reply:
I intend this to be user facing, although maybe it shouldn't be advertised.